wasm-tests/Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
```

Comment:
can these wasm-tests bits be moved into the `testing` folder?

Reply:
sure